rex/renewable_resource.py

```diff
@@ -1162,14 +1163,17 @@ def preload_SAM(cls, h5_path, sites, unscale=True, str_decode=True,
             e.g. '5min'.
         means : bool
             Boolean flag to compute mean resource when res_array is set
+        check_files : bool
+            Check to ensure files have the same coordinates and time_index
 
         Returns
         -------
         SAM_res : SAMResource
             Instance of SAMResource pre-loaded with Solar resource for sites
             in project_points
         """
-        with cls(h5_path, unscale=unscale, str_decode=str_decode) as res:
+        with cls(h5_source, unscale=unscale, str_decode=str_decode,
+                 check_file=check_file) as res:
```

Comment:
kwarg is check_file (singular) here and check_files (plural) below in MultiFileWTK

Reply:
I knew I was going to do that...